### PR TITLE
docs(mesh) Formatting and image name fixes for installation

### PR DESCRIPTION
**Pull request by lena-larionova**
* Adjusting formatting, language, grammar
* For Docker install: fixing Docker image name to point to Kong Mesh instead of Kuma (tested pulling them, they should be correct now)
* Adding tabs to installation pages that were missing them to match other installation pages
* Removing link to Helm and Helm chart tab, as Helm install doesn't exist yet. Will re-add when that changes.

Previews:
- [Getting Started](https://deploy-preview-2290--kongdocs.netlify.app/mesh/1.0.x/gettingstarted/) - included this topic because it's mainly about installation
- [Install](https://deploy-preview-2290--kongdocs.netlify.app/mesh/1.0.x/install/) - see all installation topics under this section

### CODE REVIEW

File: app/mesh/1.0.x/install.md [hunk 1]
```diff
@@ -5,9 +5,12 @@ no_search: true
 
 ## Install {{site.mesh_product_name}}
 
-{{site.mesh_product_name}} is built on top of Kuma and Envoy and in order to create a seamless experience it follows the same exact installation and configuration procedures as Kuma, but with its own binaries instead.
+{{site.mesh_product_name}} is built on top of Kuma and Envoy. To create a
+seamless experience, {{site.mesh_product_name}} follows the same installation
+and configuration procedures as Kuma, but uses its own binaries.
 
-On this page you will find access to the official {{site.mesh_product_name}} distributions that provide a drop-in replacemement to Kuma's native binaries.
+On this page you will find access to the official {{site.mesh_product_name}}
```
Comment (tlblessing) [inline app/mesh/1.0.x/install.md:12]:
```suggestion
On this page, you will find access to the official {{site.mesh_product_name}}
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
"With" is a preposition and should not be capitalized (global suggested edit; e.g. Kong Mesh with Ubuntu). https://www.grammarly.com/blog/capitalization-in-the-titles/

A few commas missing. Suggested an edit and screenshot here since the text not in PR:
image

**Comment by lena-larionova** [comment]
> "With" is a preposition and should not be capitalized (global suggested edit; e.g. Kong Mesh with Ubuntu). https://www.grammarly.com/blog/capitalization-in-the-titles/

You're right - I missed looking at the titles completely. 

> A few commas missing.

Chalk it up to editing fatigue. Will look over the install topics again.

**Review by tlblessing — APPROVED** [review]
LGTM

**Pull request merged**